assets/js/components/Layout/Layout.jsx

```diff
@@ -73,14 +73,46 @@ const Layout = () => {
         <div className="flex flex-col h-screen items-start justify-between">
           <div
             className={classNames(
-              'h-screen block shadow-lg fixed w-64 flex-shrink-0 z-20',
-              { hidden: isCollapsed }
+              'h-screen block shadow-lg fixed flex-shrink-0 z-20',
+              { 'w-16': isCollapsed, 'w-64': !isCollapsed }
             )}
           >
+            <div
+              className={classNames('block absolute mt-4', {
+                'ml-64': !isCollapsed,
```

Comment:
You could have this like:
```
isCollapsed? 'ml-16' : 'ml-64'
```

Reply:
Actually it's more idiomatic this @arbulu89 :sweat_smile: sorry about that, I know we have the other way around repeated a lot and really scattered around the code, but it was also something that we wrote prior to `classnames` advent in this codebase :/

Reply:
@arbulu89 Thanks for the recommendation. After reviewing both suggestions, I have decided to follow @dottorblaster suggestion here.